libraries/chem-meta/src/rdkit-api.ts

```diff
@@ -14,6 +14,8 @@ export interface RDModule {
   _malloc(size: number): any;
   _free(buf: any): any;
```

Comment:
Does it really accept and return "any"?

Reply:
Same as previous comment. Function will be removed.